Recruit-Training/Telemetry-Recruit-Training/README.md

```diff
@@ -0,0 +1,36 @@
+= U of C Solar Car Telemetry Recruit Training
+
+Intro project to learn about the telemetry server 
+
+== Running the project
+
+=== Via Command Line
+
+To run this project locally, navigate to the source directory and run the following command:
+
+[source,bash]
+----
+ng serve
+----
+
+Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
+
+== Objective
+
+This project will give recruits a introduction to how the telemetry server works.
+By completing this training, recruits can develop some skills that are required to work on the server.
+
+=== Tasks to Complete
+
+==== Create a MPPT tab and component
+
+ *  Run `ng generate component mppt` in the command line to generate a new mppt component within the tabs folder.
```

Comment:
If you're giving more detailed instructions like this, do you think it would be good to include the part about routing?